plugins/org.eclipse.fordiac.ide.typeeditor/src/org/eclipse/fordiac/ide/typeeditor/AbstractTypeEditor.java

```diff
@@ -208,21 +205,12 @@ public void dispose() {
 			annotationModel.dispose();
 		}
 
-		// get these values here before calling super dispose
```

Comment:
Please revert these changes. With the current overall 4diac IDE typelib this can not yet be removed.